run.py

```diff
@@ -411,6 +414,52 @@ def run(args):
                            docker_tag=docker_tag))
         service.start_launch(name=launch_name, start_time=timestamp(), description=launch_desc)
 
+    tests = suite.get('tests')
+    res = []
+    for test in tests:
+        test = test.get('test')
+        tc = dict()
+        tc['name'] = test.get('name')
+        tc['desc'] = test.get('desc')
+        tc['file'] = test.get('module')
+        tc['rhbuild'] = rhbuild
+        tc['ceph-version'] = ceph_version
+        tc['ceph-ansible-version'] = ceph_ansible_version
+        tc['compose-id'] = compose_id
+        tc['distro'] = distro
+        tc['suite-name'] = suite_name
+        tc['suite-file'] = suite_file
+        tc['conf-file'] = glb_file
+        tc['ceph-version-name'] = ceph_name
+        test_file = tc['file']
+        report_portal_description = tc['desc'] or ''
+        test_names.append(tc['name'])
+        tc['log-link'] = None
+        tc['duration'] = '0s'
+        tc['status'] = 'Not Executed'
```

Comment:
`tc` is list which looks like a repetive code. place it in one location. so that the same function will be used before and after the tests.

Reply:
Made the requested changes. Added a method that collects all the test details and adds them to the dictionary and returns it.